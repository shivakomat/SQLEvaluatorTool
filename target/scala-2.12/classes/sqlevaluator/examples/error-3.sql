-- Type mismatch: "name" is a str and "population" is an int.
-- Comparisons only work between values of the same type.
SELECT name
FROM cities
WHERE name = "z" AND name != population
