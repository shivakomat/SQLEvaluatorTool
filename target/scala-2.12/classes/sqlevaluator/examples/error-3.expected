ERROR: Incompatible types to "!=": str and int.
